desc/io/efit_io.py

```diff
@@ -0,0 +1,234 @@
+"""Utilities for reading EFIT geqdsk files."""
+
+import numpy as np
+from matplotlib import pyplot as plt
+from scipy.interpolate import CubicSpline
+
+
+def efit_to_desc(g, M, N=0, L=None, sep_dev=0):
+    """Create a DESC equilibrium from an EFIT geqdsk file.
+
+    The equilibrium will not be solved, this just creates an equilibirum
+    with the correct boundary and profiles.
+
+    Parameters
+    ----------
+    g : path-like or dict
+        Path to geqdsk file, or dict of already read parameters.
+    M : int
+        Poloidal resolution of the equilibrium. Used for fitting the boundary.
+    N : int, optional
+        Toroidal resolution of the equilibrium.
+    L : int, optional
+        Radial resolution of the equilibrium. Used for fitting profiles.
+        If None, spline profiles will be used. Otherwise they are fit
+        to even power series in rho.
+    sep_dev : float
```

Comment:
sep_dev is not very intuitive, is this a usual term in the tokamak community? 

I think something more like `fraction_of_bdry` or something better where it defualts to `1.0` if the separatrix is desired is better, especially since you can ask for `0.95` and get the `q95` surface which is (I think) used a lot in cases where a hard separatrix is not desired.

Reply:
Ok, I'll do that.